src/main/java/org/haffson/adventofcode/days/day01/Day01.java

```diff
@@ -65,4 +67,22 @@ private int calculateFrequency(List<Integer> myIntArrayList) {
                 .mapToInt(Integer::intValue)
                 .sum();
     }
+
+    private int calculateFrequencyPart2(List<Integer> myIntArrayList){
+        int sum = 0;
+        HashSet<Integer> found = new HashSet<>();
```

Comment:
Zur Benennung der Variablen: Vielleicht "found" in "previousSums" oder ähnlich umbenennen. Und "sum" ist ja die Summierung über alle Durchläufe. Keine Ahnung welcher Name schön ist, aber auch da kann ein guter Name Dritten (wie mir 😢  ) beim Verstehen helfen.

Reply:
Guter Hinweis - direkt mal angepasst :)